pkg/security/secl/rules/ruleset.go

```diff
@@ -648,12 +648,12 @@ func (rs *RuleSet) Evaluate(event eval.Event) bool {
 				rs.logger.Tracef("Rule `%s` matches with event `%s`\n", rule.ID, event)
 			}
 
-			rs.NotifyRuleMatch(rule, event)
-			result = true
-
 			if err := rs.runRuleActions(event, ctx, rule); err != nil {
 				rs.logger.Errorf("Error while executing rule actions: %s", err)
 			}
+
+			rs.NotifyRuleMatch(rule, event)
+			result = true
```

Comment:
Not sure this will reassure but I don't think that I know better than you :-)
I think for the 2 actions that we currently support, it makes sense to execute the actions before sending the event.

Reply:
Yep I agree, thanks!